config/README.md

```diff
@@ -0,0 +1,31 @@
+# Configuration Library
+
+This package can be used to parse a configuration file that follows the JSON
+Schema defined by the [OpenTelemetry Configuration] schema.
+
+The package contains:
+
+- models generated via the JSON schema using the [go-jsonschema] library
```

Comment:
Are we not we concerned that we are coupling the model to the JSON, YAML parsing libraries?
Reference: https://www.reddit.com/r/golang/comments/rtne10/good_antipattern_blog_post/